charts/external-es-proxy/templates/external-es-proxy-configmap.yaml

```diff
@@ -0,0 +1,68 @@
+################################
+## Elasticsearch Proxy ConfigMap
+################################
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "external-es-proxy.fullname" . }}
+  labels:
+    tier: external-logging
+    component: {{ template "external-es-proxy.name" . }}
+    chart: {{ template "external-es-proxy.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+data:
+  nginx.conf: |-
+    worker_processes 1;
+    pid /tmp/nginx.pid;
+    events { worker_connections 1024; }
+    error_log /dev/stdout info;
+    env ES_SECRET;
+    env ES_SECRET_NAME;
+
+    http {
+      access_log /dev/stdout;
+      client_max_body_size 1024M;
+
+      server {
+        listen 9200;
+        auth_request /auth;
+        proxy_http_version 1.1;
+        {{- if not .Values.global.istio.enabled }}
+        proxy_set_header Host $host;
+        {{- end }}
+        proxy_set_header Connection "Keep-Alive";
+        proxy_set_header Proxy-Connection "Keep-Alive";
+
+        location ~ ^/ {
+          access_by_lua_file  /usr/local/openresty/nginx/conf/setenv.lua;
+          proxy_pass {{.Values.global.custom_logging.scheme}}://{{.Values.global.custom_logging.host}}:{{.Values.global.custom_logging.port}};
```

Comment:
Could we include a mechanism for passing and configuring a self signed certificate authority as an option? A lot of customers that this applies to would likely have private ca certs that they use for their own infrastructure.

Reply:
yes sure, i didn't verify with self-signed/private certs.let me add necessary changes .

Reply:
Seems like this conversation can be resolved?

Reply:
Yes we now support trust certs as well